.github/workflows/base_builder.yaml

```diff
@@ -2,7 +2,7 @@ name: Build AnalysisBase base image
 on:
   push:
     branches:
-      - main
+      - "*"
```

Comment:
No, this should stay `main`. We don't want to deploy on PRs.

Reply:
I need to be able to test things and it would be really helpful if we could have other branches also build images, just tag them with the branch name. And also derived images tagged with the same branch name.
Don't you think so?